pkgs/development/python-modules/taxi/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, appdirs
+, requests
+, click
+, setuptools
+, backends ? [ ]
+}:
+
+buildPythonPackage rec {
+  pname = "taxi";
+  version = "6.1.1";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "taxi";
+    sha256 = "b2562ed58bd6eae7896f4f8e48dbee9845cd2d452b26dd15c26f839b4864cb02";
+  };
+
+  # No tests in pypy package
+  doCheck = false;
```

Comment:
Then the package must be fetched from GitHub where there are tests. https://github.com/sephii/taxi/tree/main/tests

Also a pythonImportsCheck is missing which leaves the package with zero testing if it could actually work.

Reply:
Ok I'll improve this aspect and resubmit a PR. Thanks for tips.